### ISSUE TITLE
Banjo-Tooie Video locks up on intro

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#237]
Originally reported on Google Code with ID 238

```
Describe your system:
 - Mac OS X: 10.5.7
 - Machine type: 64-bit (32-bit capable)
 - Mupen64Plus version: 1.5 Beta
 - Plugins used: tried with all graphics plugins (Rice Video 1.5-development, Glide64

Mupen64Plus, gln64)

For visual artifacts, give:
 - Banjo-Tooie (U) [!].v64, video locks during intro

Describe the problem:

I run the game choosing the first save game when it asks. It goes through some of the
intro. 
After Banjo said: "perhaps one of us should go take a look outside" The video completely
locks 
up on a black screen, but the emulation itself continues, so you still hear the sound.

```

Reported by `agsgymboy@hotmail.com` on 2009-06-02 22:27:44

**Comment by richard42** [comment]
```
update: the dialog sounds stop, but the background music continues.
```

Reported by `agsgymboy@hotmail.com` on 2009-06-02 22:30:20

**Comment by richard42** [comment]
```
Did you try disabling dynarec? this is known to cause random crashes in this game
```

Reported by `auria.mg` on 2009-06-04 00:47:34

**Comment by richard42** [comment]
```
how do you turn off dynarec?
```

Reported by `agsgymboy@hotmail.com` on 2009-06-04 04:41:16

**Comment by richard42** [comment]
```
alright, even with the dynamic recompiler (dynarec) off, and having it switched to
interpreter, or even pure interpreter, it still crashes in the same area,
specifically ugga wuggas cave (or whatever it is called) in the first level. So i
don't think that the Dynarec is the problem (even though other people say it is).
Please Mupen64 staff, FIX THIS!!!!!!! i want to be able to play banjo tooie without
it locking up IN THE FIRST LEVEL!!!!!!!!!!
```

Reported by `agsgymboy@hotmail.com` on 2009-06-05 02:08:02

**Comment by richard42** [comment]
```
You know that you want a game to run on non-licensed platform.

Anyway. Can you reproduce it at special positions? So it would be nice if you could
create a snapshot right before a position where it crashes. So for example if it
crashes inside a dungeon then please make a snapshot right before the dungeon so that
a developer only has to press forward to create the crash.
Please attach this snapshot or upload somewhere so a developer can inspect the
problem. Don't forget the md5 checksum of the rom.

Have you tried a new version from svn?
```

Reported by `sven@narfation.org` on 2009-09-21 15:13:06

**Comment by richard42** [comment]
```
It seems that the Unga Wunga part is a duplicate of
http://code.google.com/p/mupen64plus/issues/detail?id=101
```

Reported by `sven@narfation.org` on 2009-09-21 20:12:06

**Comment by richard42** [comment]
```
Merging this into issue 101.
```

Reported by `richard42g` on 2010-01-12 21:58:12
- **Status changed**: `Duplicate`
- **Labels added**: Type-Defect, Component-Core
- **Merged into**: #101
